Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
 
 gem 'hyperresource'
 
+gem 'addressable'
```

Comment:
It can parse the query params from a uri string into a hash:
`URI.parse(uri).query` returns `id=12345`, but `Addressable::URI.parse(uri).query_values` returns`{id: 12345}`